source/extensions/filters/common/expr/context.cc

```diff
@@ -141,15 +143,18 @@ absl::optional<CelValue> ResponseWrapper::operator[](CelValue key) const {
   } else if (value == Flags) {
     return CelValue::CreateInt64(info_.responseFlags());
   } else if (value == GrpcStatus) {
-    auto const& optional_status =
-        Grpc::Common::getGrpcStatus(*(trailers_.value_), *(headers_.value_), info_);
+    ConstSingleton<Http::HeaderMapImpl> empty_headers;
```

Comment:
You don't need even to instantiate `ConstSingleton` here. Just use `ConstSingleton<Http::HeaderMapImpl>::get()`.

Reply:
Ah yeah, it's a static member. Done.